config/channel/webhook/webhook-deployment.yaml

```diff
@@ -56,8 +56,15 @@ spec:
           containerPort: 9090
         - name: profiling
           containerPort: 8008
-        # TODO set proper resource limits.
-
+        resources:
+          requests:
+            # taken from serving.
+            cpu: 100m
+            memory: 50Mi
+          limits:
+            # taken from serving.
+            cpu: 200m
+            memory: 200Mi
```

Comment:
I know this was taken from serving (and from there to eventing as well).
But something to have in mind that in "real world" we usually want larger value.

for Openshift Serverless, we do override that (in the extensions for serving and eventing) to 1G:
https://github.com/openshift-knative/serverless-operator/blob/main/openshift-knative-operator/pkg/eventing/extension.go#L55-L56

Reply:
sure -- I can change the memory limit to 1G, anything else you'd prefer changed value wise?